dolphinscheduler-python/pydolphinscheduler/tests/core/test_task.py

```diff
@@ -47,14 +47,14 @@
         (
             {
                 "local_params": ["foo", "bar"],
-                "resource_list": ["foo", "bar"],
+                "resource_list": [{"id": 1}, {"id": 2}],
                 "dependence": {"foo", "bar"},
                 "wait_start_timeout": {"foo", "bar"},
                 "condition_result": {"foo": ["bar"]},
             },
             {
                 "localParams": ["foo", "bar"],
-                "resourceList": ["foo", "bar"],
+                "resourceList": [{"id": 1}, {"id": 2}],
```

Comment:
I find out we just add style `[{"key": val}, {"key": val}]` for test, can we also add `["foo", "bar"]` type resource for test and mock the function `query_resource` return values? I am not sure whether we can add this case to this function or not, but if we can not, please add new test function to cover it